OpenRA.Mods.RA/AI/HackyAI.cs

```diff
@@ -310,7 +310,7 @@ public bool HasMinimumProc()
 		// For mods like RA (number of building must match the number of aircraft)
 		bool HasAdequateAirUnits(ActorInfo actorInfo)
 		{
-			if (!actorInfo.Traits.Contains<ReloadsInfo>() && actorInfo.Traits.Contains<LimitedAmmoInfo>()
+			if (actorInfo.Traits.Contains<LimitedAmmoInfo>() 
```

Comment:
This method used to check if the actor needs an airfield to reload and if there are enough airfields for all actors of this type.
You need to add a check for `!ReloadsAutomatically()' to stay true to the idea of the method.